libs/api/prisma/schema.prisma

```diff
@@ -123,14 +123,33 @@ model Article {
   hidden          Boolean @default(false)
   disableComments Boolean @default(false)
 
-  navigations NavigationLink[]
-  tags        TaggedArticles[]
+  navigations    NavigationLink[]
+  tags           TaggedArticles[]
+  trackingPixels ArticleTrackingPixels[]
 
   @@index([createdAt])
   @@index([modifiedAt])
   @@map("articles")
 }
 
+enum TrackingPixelProviderType {
+  prolitteris @map("prolitteris")
+}
+
+model ArticleTrackingPixels {
+  id                        String                    @id @default(cuid())
+  article                   Article                   @relation(fields: [articleId], references: [id], onDelete: Cascade)
+  articleId                 String
+  createdAt                 DateTime                  @default(now())
+  modifiedAt                DateTime                  @updatedAt
+  trackingPixelProviderID   String
```

Comment:
I would move the trackingPixelProvider into a separate table. I am missing normalization here.

Reply:
If we want to add further fields later, it would really help. Maybe we would like to add a name or a description. Or even configurations...